Cartfile

```diff
@@ -7,3 +7,7 @@ github "ps2/rileylink_ios" "merge-from-tidepool"
 github "LoopKit/Amplitude-iOS" "decreepify"
 github "LoopKit/MKRingProgressView" "appex-safe"
 github "dexman/Minizip" "master"
+
+# Firebase Crashlytics
+binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseAnalyticsBinary.json"
```

Comment:
Do we need the analytics here? It looks like the Firebase iOS SDK is open source, but the Firebase iOS SDK repo does not include analytics? Is the analytics part closed source? It also has *11* frameworks in it. Really feels like bloatware.

Reply:
Just checked, and the 12 frameworks (11 from analytics and 1 from crashlytics) are 45M. Loop.app is currently 108M, so this would be increasing the size of the app by 40%, just to track crashes, in an app that already uploads crashes to apple.

Reply:
Yes, I know there are a lot of dependencies.  Note: (1) the app doesn't _automatically_ upload crashes to Apple, and (2) we have no process for addressing those crashes promptly.  I think Crashlytics is our best bet.

Reply:
I'm guessing we'll be using less than 5% of the code this PR pulls in.

Reply:
Unfortunately it is a requirement to include `FirebaseAnalyticsBinary` according to the instructions. (See the PR Description for references)